src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/multi-cluster/multi-cluster-form/multi-cluster-form.component.html

```diff
@@ -0,0 +1,150 @@
+                   type="text"
+                   formControlName="username">
+              <span class="invalid-feedback"
+                    *ngIf="remoteClusterForm.showError('username', frm, 'required')"
+                    i18n>This field is required.
+              </span>
+          </div>
+        </div>
+        <div class="form-group row"
+             *ngIf="!remoteClusterForm.getValue('showToken')">
+          <label class="cd-col-form-label required"
+                 for="password"
+                 i18n>Password
+          </label>
+          <div class="cd-col-form-input">
+            <input id="password"
+                   name="password"
+                   class="form-control"
+                   type="password"
+                   formControlName="password">
+              <span class="invalid-feedback"
+                    *ngIf="remoteClusterForm.showError('password', frm, 'required')"
+                    i18n>This field is required.</span>
+          </div>
+        </div>
+        <div class="form-group row">
+          <div class="cd-col-form-offset">
+            <div class="custom-control custom-checkbox">
+              <input class="custom-control-input"
+                     id="showToken"
+                     type="checkbox"
+                     (click)="showToken = !showToken"
+                     formControlName="showToken"
+                     [readonly]="true">
+              <label class="custom-control-label"
+                     for="showToken"
+                     i18n>Auth with token</label>
+            </div>
+          </div>
+        </div>
+        <div class="form-group row"
+             *ngIf="remoteClusterForm.getValue('showToken')">
+          <label class="cd-col-form-label required"
+                 for="apiToken"
+                 i18n>Token
+          </label>
+          <div class="cd-col-form-input">
+            <input id="apiToken"
+                   name="apiToken"
+                   class="form-control"
+                   type="text"
+                   formControlName="apiToken">
+            <span class="invalid-feedback"
+                  *ngIf="remoteClusterForm.showError('apiToken', frm, 'required')"
+                   i18n>This field is required.</span>
+          </div>
+        </div>
+        <div class="form-group row">
+          <div class="cd-col-form-offset">
+            <div class="custom-control">
+              <button class="btn btn-primary"
+                      type="button"
+                      [disabled]="(remoteClusterForm.getValue('showToken') && remoteClusterForm.getValue('apiToken') === '') || (!remoteClusterForm.getValue('showToken') && (remoteClusterForm.getValue('username') === '' || remoteClusterForm.getValue('password') === ''))"
+                      (click)="verifyConnection()">
+                Verify Connection
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
```

Comment:
I don't like the idea of a separate verify connection button. The `connect cluster` implies that, the connection will be verified as part of that action. So why do we need to do an extra step and force user to click this button?

Instead, when we click the submit button, it'll already check to see if connection can be established. If no connection is established, just trigger an error in the form and display it (either in the form or in notification). I think that would be more than fine.

Moreover, as part of verification, you are actually sending the login request, which means that the same request that we sent when we do Connect Cluster is being used for Verify Connection which duplicates things for unnecessary things.